docs/vm-ee-guide.md

```diff
@@ -0,0 +1,267 @@
+# Onboard
+
+## Table of Contents
+
+<!-- toc -->
+- [VirtualMachines](#virtualmachines)
```

Comment:
It should be Virtual Machine and External Entity with space

Reply:
Done